### DISCUSSION TITLE
Mesh resolution validation

### DISCUSSION THREAD

**Discussion by navramesh** [firemodels/fds#13387] [Q&A]
hi everyone,
Just a query about the validation of the Mesh resolution we choose to run a fds simulation. In the earlier versions of the fds user guide, I think up to the late 6.8 versions, the mesh resolution was validated by calculating the plume resolution index. It was suggested that the ratio be between 4 to 16. But the same statement has not been suggested in the latest version. Does it mean that there is no particular validation technique other than run a mesh sensitivity study? could someone please clarify this?
Thank you.

**Comment by drjfloyd** [comment]
Earlier versions of the User's Guide noted the plume length scale and the range of values used at that time in the Validation Guide. In earlier versions, there was a much more limited set of validation cases than in the current version. In the current Validation Guide there is a very large range of D*/dx values (1 to 118) some of which may not be appropriate for some applicaitons (which was also the case previously). The language in the User's Guide has been changed to reflect this.

**Reply by navramesh** [reply]
Ok thank you so much for the clarification.
